cluster/addons/volumesnapshots/volume-snapshot-controller/rbac-volume-snapshot-controller.yaml

```diff
@@ -53,7 +53,6 @@ metadata:
   name: volume-snapshot-controller-role
   namespace: kube-system
   labels:
-    kubernetes.io/cluster-service: "true"
```

Comment:
Why removing this?

Reply:
@xing-yang iic, this label has been deprecated for addon manager. 
https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/README.md
Additionally, we do have `addonmanager.kubernetes.io/mode: Reconcile`  label which does the reconcilation. 
Does that assumption correct ? or did I miss any?

Reply:
Can you add this in the PR description section?

Reply:
Sure .. let me add the same!